README.md

```diff
@@ -1,5 +1,7 @@
-# Google Cloud Platform Codelabs
+# Google Cloud Node.js Codelab
 
-This repository contains the source material for codelabs released at Google Cloud Platform Events. [Access the codelabs](https://gcplab.me/).
+This repository contains the source code for the Google Cloud Node.js Codelab.
```

Comment:
Nit: should we add a more detailed description of what this codelab is about? e.g. *Google Cloud **Getting started with** Node.js codelab*.

Reply:
Fix applied.